runtime/compiler/runtime/codertinit.cpp

```diff
@@ -219,7 +219,9 @@ J9JITConfig * codert_onload(J9JavaVM * javaVM)
       }
 
    /* Should use portlib */
-#if defined(TR_HOST_X86)
+#if defined(TR_HOST_POWER)
+   jitConfig->codeCacheAlignment = 64;
```

Comment:
I suggest we add a comment, similar to Z, on why this value was chosen. Sure someone can use `git blame` and trace back the history. But as files move around, get renamed, refactored, etc. these things can be hard to track. It is more informative to just put a short comment right in the code so future readers will immediately understand why the alignment is needed, and specifically a 64-byte alignment.

Reply:
Done. Would you like to see comments added on the X86 32-byte alignment? To my knowledge, this was to align to a 32-byte cache line on earlier processors.

Reply:
Sure! If you've dug up on the history and that is the case I think that would be very helpful to future readers. Thanks for making the change!